code/src/main/webapp/JavaScript/script.js

```diff
@@ -179,17 +179,14 @@ function attachSignin(element, negativeUser) {
     localStorage.setItem('idToken', idToken.toString());
     const params = new URLSearchParams()
     params.append('idToken', idToken);
-    var servlet = "";
-    if (negativeUser) {
-      servlet = '/get-negative-user-info';
-    } else {
-      servlet = '/get-positive-user-info';
-    }
-
-    fetch(new Request(servlet, {method: 'POST', body: params}))
-    .then(response => response.text())
-    .then(url => window.location = url);
-
+    params.append('systemMessage', 'VERSION_1');
+    params.append('localityResource', 'US');
+    params.append('messageLanguage', 'SP');
+    console.log("here");
+    console.log(params);
+    fetch(new Request('/send-messages', {method: 'POST', body: params})).then(console.log("cool"));
+    // url => window.location = url
+    console.log("cool2");
```

Comment:
what's happening here?

Reply:
Don't know I think this is from the send-msg-pull branch